definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -0,0 +1,167 @@
+    "CaseFieldID": "D8DerivedRespondentSolicitorAddr",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "D8DerivedRespondentHomeAddress",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "D8DerivedRespondentCorrespondenceAddr",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "DerivedCoRespondentSolicitorAddr",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "D8DerivedReasonForDivorceAdultery3rdAddr",
+    "PageFieldDisplayOrder": 5,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "BulkListingCaseId",
+    "PageFieldDisplayOrder": 6,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "tempEditDerivedAddresses",
+    "CaseFieldID": "PreviousCaseId",
+    "PageFieldDisplayOrder": 7,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "tempEditDerivedAddresses",
+    "PageLabel": "Temporary - Edit derived addresses",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Are you sure these labels should be in prod? It looks like a strange thing in a clean-up PR.

Reply:
no, this file should be 

```
[]
```